x/dispensation/keeper/executors_test.go

```diff
@@ -63,7 +63,7 @@ func TestKeeper_DistributeDrops_For_Address_Fail(t *testing.T) {
 	runner := ""
 	err = keeper.CreateDrops(ctx, outputList, distributionName, types.DistributionType_DISTRIBUTION_TYPE_AIRDROP, runner)
 	assert.NoError(t, err)
-	_, err1 := keeper.DistributeDrops(ctx, 4657424885079777562, distributionName, runner, types.DistributionType_DISTRIBUTION_TYPE_AIRDROP)
+	_, err1 := keeper.DistributeDrops(ctx, 4657424885079777562, distributionName, runner, types.DistributionType_DISTRIBUTION_TYPE_AIRDROP, 10)
```

Comment:
should these lines with 10 instead refer to a constant variable, say DEFAULT_DISTRIBUTION_COUNT?

Reply:
We don't have a DEFAULT_DISTRIBUTION_COUNT under the design , the number 10 is just the value we had earlier, to help the tests pass.
There is going to be a separate PR with unit tests, ill try to include the changes related to tests in that